proto/interchain_security/ccv/v1/shared_consumer.proto

```diff
@@ -0,0 +1,146 @@
+syntax = "proto3";
+
+package interchain_security.ccv.v1;
+
+option go_package = "github.com/cosmos/interchain-security/v3/x/ccv/types";
+
+import "tendermint/abci/types.proto";
+import "ibc/lightclients/tendermint/v1/tendermint.proto";
+import "ibc/core/channel/v1/channel.proto";
+import "google/protobuf/duration.proto";
+import "gogoproto/gogo.proto";
+import "interchain_security/ccv/v1/wire.proto";
+import "google/protobuf/timestamp.proto";
+
+//
+// Note any type defined in this file is referenced/persisted in both the consumer and provider CCV modules,
+// but not sent over the wire. These schemas could change, only with careful consideration of effects!
+//
+
+// TODO: rename to ConsumerParams (make issue)
```

Comment:
We'd need to rename the consumer `Params` type to `ConsumerParams`, since it's no longer in the x/ccv/consumer package. now it's in x/ccv